spec/models/game_spec.rb

```diff
@@ -74,4 +74,61 @@
       expect(game.active_player).to eq(game.black_player)
     end
   end
+
+  describe '#stalemate?' do
+    before do
+      allow_any_instance_of(Game).to receive(:populate_board!).and_return true
+    end
+
+    let(:game) { FactoryGirl.create(:game) }
+    let(:white_king) { FactoryGirl.create(:king, type: 'King', color: 'white', row: 7, column: 3) }
+    let(:black_king) { FactoryGirl.create(:king, type: 'King', color: 'black', row: 0, column: 3) }
```

Comment:
I think you can add `game: game` to both `white_king` and `black_king` then you won't need to add it to game in each of the tests.